_rules/inline_link_in_paragraph_is_distinguishable_be4d0c.md

```diff
@@ -0,0 +1,252 @@
+---
```

Comment:
Can you change the file name to use `-` instead of `_`? That's what we've used in all other rules.